pkg/chunk/object/chunk_object_client.go

```diff
@@ -0,0 +1,80 @@
+package object
+
+import (
+	"bytes"
+	"context"
+	"io/ioutil"
+
+	"github.com/cortexproject/cortex/pkg/chunk"
+	"github.com/cortexproject/cortex/pkg/chunk/util"
+)
+
+// ChunkClient is used to store chunks in object store backends
+type ChunkClient struct {
+	Store chunk.ObjectClient
```

Comment:
Is `Store` required to be exposed?

Reply:
No, I’ll make it private and add a constructor function